coreneuron/nrniv/cellorder1.cpp

```diff
@@ -531,9 +532,9 @@ static void admin1(int ncell,
     // cellsize is the number of nodes in the cell not counting root.
     // nstride is the maximum cell size (not counting root)
     // stride[i] is the number of cells with an ith node.
-    firstnode = new int[ncell];
-    lastnode = new int[ncell];
-    cellsize = new int[ncell];
+    firstnode = (int*)ecalloc_align(ncell, NRN_SOA_BYTE_ALIGN, sizeof(int));
+    lastnode = (int*)ecalloc_align(ncell, NRN_SOA_BYTE_ALIGN, sizeof(int));
+    cellsize = (int*)ecalloc_align(ncell, NRN_SOA_BYTE_ALIGN, sizeof(int));
```

Comment:
have you done clang-format? you have to:
* install clang-format 4.0
* cmake ..
* make formatsource

This will format the code consistently.

Reply:
Sure, I'll do it once I'll finish applying these changes.